benches/benches_agate_rocks.rs

```diff
@@ -0,0 +1,253 @@
+#![cfg(feature = "enable-rocksdb")]
+mod common;
+
+use std::{
+    ops::Add,
+    sync::Arc,
+    time::{Duration, Instant},
+};
+
+use agatedb::{Agate, AgateOptions, IteratorOptions};
+use common::{gen_kv_pair, remove_files, unix_time};
+use criterion::{criterion_group, criterion_main, Criterion};
+use rand::{prelude::ThreadRng, Rng};
+use rocksdb::DB;
+use tempdir::TempDir;
+
+const BATCH_SIZE: u64 = 1000;
+const SMALL_VALUE_SIZE: usize = 32;
+const LARGE_VALUE_SIZE: usize = 102400;
+
+fn agate_populate(agate: Arc<Agate>, value_size: usize) {
+    let mut txn = agate.new_transaction_at(unix_time(), true);
+
+    for i in 0..BATCH_SIZE {
+        let (key, value) = gen_kv_pair(i, value_size);
+        txn.set(key, value).unwrap();
+    }
+
+    txn.commit_at(unix_time()).unwrap();
+}
+
+fn agate_randread(agate: Arc<Agate>, value_size: usize, rng: &mut ThreadRng) {
```

Comment:
can we reuse the logic of agate bench and support multiple threads

Reply:
I have updated this, PTAL.